packages/api-server/src/middleware/withWebServer.ts

```diff
@@ -6,11 +6,21 @@ import type { Application } from 'express'
 
 import { getPaths } from '@redwoodjs/internal'
 
+type Contents = {
+  [content: string]: string
+}
 const withWebServer = (app: Application) => {
-  const indexContent = fs.readFileSync(
-    path.join(getPaths().web.dist, '/index.html'),
-    'utf-8'
-  )
+  const files = fs.readdirSync(getPaths().web.dist)
+    .filter(file => file.split('.')[1] === 'html')
+    .map(file => file.split('.')[0])
```

Comment:
Could we use `path.basename` here? https://nodejs.org/api/path.html#path_path_basename_path_ext